src/content/en/tools/workbox/guides/advanced-recipes.md

```diff
@@ -87,6 +81,17 @@ window.addEventListener('load', function() {
       return;
     }
 
+    // When the user asks to refresh the UI, we'll need to reload the window
+    var refreshing;
+    navigator.serviceWorker.addEventListener('controllerchange', function(event) {
+      // Ensure refresh is only called once.
+      // This works around a bug in "force update on reload".
+      if (refreshing) return;
```

Comment:
Can you rename `refreshing` to `window.__preventDevToolsReloadLoop`, this just makes it clear what the boolean is doing and the double underscore just makes it clear that it's a custom property.

Reply:
Can you say a little more about why? This local `refreshing` variable doesn't need to be shared globally; generally speaking, it's better to use local variables instead of putting stuff on the global `window` object.

Reply:
At the moment it's ambiguous what you are trying to assign this to. I don't mind if assigning it to the function works, just assign it to something.

Reply:
> it's ambiguous what you are trying to assign this to

I think we may be miscommunicating. I'm not "trying to assign" `refreshing` at all at line 85.

Specifically, I think you may have incorrectly assumed that `var refreshing` on line 85 is assigning refreshing to something, either as a property of the `window` or as a property of a function.

That's not what `var` does here. `var refreshing` defines a local variable, scoped to current function; it's not an assigned property of the window, the function, or anything else. As a local variable, it's only visible inside this function, which is much safer/better than assigning it somewhere visible (where it could clobber someone's existing variable).

For example, even if there _were_ a `window.refreshing` variable, `var refreshing` here in this function would shadow the `refreshing` variable on the global scope, automatically preventing a naming conflict.

In an effort to meet you halfway, I've renamed the variable, so now it says `var preventDevToolsReloadLoop`, but with no underscores and no explicit assignment, ensuring that it can't conflict with global window properties.

I hope this explanation makes sense and the updated name will be agreeable. LMK if this didn't make sense.